### ISSUE TITLE
Unexpected token j in JSON at position 4787

### ISSUE THREAD

**Issue by fengyiyingdong** [interfaced/zombiebox#10]
Meet exception, how can I to solve it, thanks.

```
Loaded extension cutejs
Loaded platform lg
Loaded platform pc
Error: Failed to parse GCC output: Unexpected token j in JSON at position 4787
java.lang.NullPointerException: NAME default$$module$Users$dwbz$iwork$github$OrsayProjectTest$node_modules$zombiebox$zb$device$input$keys 56 [length: 4] [originalname: Keys] [source_file: /Users/dwbz/iwork/github/OrsayProjectTest/node_modules/zombiebox-platform-lg/lib/input.js] : ?
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:895)
	at com.google.javascript.jscomp.RemoveUnusedCode.getVarForNameNode(RemoveUnusedCode.java:719)
	at com.google.javascript.jscomp.RemoveUnusedCode.traverseNameNode(RemoveUnusedCode.java:574)
	at com.google.javascript.jscomp.RemoveUnusedCode.traverseNode(RemoveUnusedCode.java:420)
	at com.google.javascript.jscomp.RemoveUnusedCode.access$1300(RemoveUnusedCode.java:93)
	at com.google.javascript.jscomp.RemoveUnusedCode$Continuation.apply(RemoveUnusedCode.java:1588)
	at com.google.javascript.jscomp.RemoveUnusedCode.traverseAndRemoveUnusedReferences(RemoveUnusedCode.java:269)
	at com.google.javascript.jscomp.RemoveUnusedCode.process(RemoveUnusedCode.java:250)
	at com.google.javascript.jscomp.PhaseOptimizer$NamedPass.process(PhaseOptimizer.java:317)
	at com.google.javascript.jscomp.PhaseOptimizer.process(PhaseOptimizer.java:232)
	at com.google.javascript.jscomp.Compiler.performOptimizations(Compiler.java:2431)
	at com.google.javascript.jscomp.Compiler.lambda$stage2Passes$1(Compiler.java:801)
	at com.google.javascript.jscomp.CompilerExecutor$2.call(CompilerExecutor.java:102)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by l1bbcsg** [comment]
The exception seems to arise from Google Closure Compiler failing to process some input and crashing.

Which I believe originates from `zombiebox-platform-lg` being out of sync with recent version of ZombieBox and trying to access an obsolete constant (`Keys` in `device/input/keys`).

A quick fix would be to downgrade `zombiebox`, 2.0.1 would probably be a good choice. We will however update `zombiebox-platform-lg`to be compatible with newer versions of ZombieBox shortly.

Do note however that Netcast platform is now being slowly phased out by LG and if I'm not mistaken, they are not even accepting new submissions to the app store, so developing an application for it may not be reasonable.

**Comment by l1bbcsg** [comment]
`zombiebox-platform-lg` version `2.1.0` was released to address this.

This should be fixed, feel free to reopen issue if it was not.
